### PR TITLE
Upload How2 models

### PR DESCRIPTION
**Pull request by hirofumi0810**

### CODE REVIEW

File: egs/how2/mt1/RESULTS.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # BPE 8k
```
Comment (sw005320) [inline egs/how2/mt1/RESULTS.md]: Can we reverse the order?
(better model first)
The other recipe is like that.

File: egs/how2/asr1/conf/tuning/train_pytorch_transformer_long.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# network architecture
```
Comment (sw005320) [inline egs/how2/asr1/conf/tuning/train_pytorch_transformer_long.yaml]: Can you add some comments about `_long` in this config?

File: egs/how2/asr1/RESULTS.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
 # RNN results
```
Comment (sw005320) [inline egs/how2/asr1/RESULTS.md]: Can we reverse the order?
(better model first)
The other recipe is like that.

File: egs/how2/mt1/run.sh [hunk 4]
```diff
@@ -29,8 +30,12 @@ n_average=5                  # the number of NMT models to be averaged
 use_valbest_average=true     # if true, the validation `n_average`-best NMT models will be averaged.
                              # if false, the last `n_average` NMT models will be averaged.
 
+# cascaded-ST related
```
Comment (sw005320) [inline egs/how2/mt1/run.sh]: Do we need this for MT?
(If you intend to leave these lines to make run.sh similar to ST, that is fine)

File: egs/how2/st1/RESULTS.md [hunk 5]
```diff
@@ -1,5 +1,50 @@
 # RNN results
```
Comment (sw005320) [inline egs/how2/st1/RESULTS.md]: Can we reverse the order?
(better model first)
The other recipe is like that.

File: egs/how2/st1/conf/tuning/train_pytorch_transformer_short.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
+# network architecture
```
Comment (sw005320) [inline egs/how2/st1/conf/tuning/train_pytorch_transformer_short.yaml]: what does `_short` mean?
could you explain it?

File: egs/how2/st1/conf/tuning/train_pytorch_transformer_short_long.yaml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+# network architecture
```
Comment (sw005320) [inline egs/how2/st1/conf/tuning/train_pytorch_transformer_short_long.yaml]: Same comment.
It's better to have some comments to explain `_short_long`.

**Comment by sw005320** [comment]
LGTM

File: egs/how2/mt1/run.sh (hunk 4 above)
Comment (hirofumi0810) [reply egs/how2/mt1/run.sh]: This is the minimal design for cascaded ST.

Comment (sw005320) [reply egs/how2/mt1/run.sh]: OK.

**Comment by sw005320** [comment]
I think this PR is in good shape.
If it is ready, I'll merge it.

**Comment by hirofumi0810** [comment]
> I think this PR is in good shape.
> If it is ready, I'll merge it.

@sw005320 
It's ready once CI is passed.

**Pull request merged**